OviBortas/swift-abstracted-style
# Implement code generation logic

### What does this PR do? :soccer:

This PR adds the logic needed to create the code generation for Fonts and Colors.

### Tell me about any important implementation details. :construction:


### How can this PR be tested? :mag:

1. Run `npm start` on the project.
2. Open zeplin and in the menu bar go to `Extensions` > `Manage Project Extensions`. Disable all other extensions except `swift-abstracted-style` and tap `done`
3. Go to the `Styleguide` > `Colors & Text Styles` in Zeplin
4. In the right panel where code generation is shown you should see code for named and abstracted colors generated along with abstracted fonts.
